pinot-segment-local/src/main/java/org/apache/pinot/segment/local/io/writer/impl/VarByteChunkWriter.java

```diff
@@ -28,4 +28,8 @@ public interface VarByteChunkWriter extends Closeable {
   void putString(String value);
 
   void putBytes(byte[] value);
+
+  void putStrings(String[] values);
+
+  void putByteArrays(byte[][] values);
```

Comment:
Suggest renaming them to be more clear
```suggestion
  void putStringMV(String[] values);

  void putBytesMV(byte[][] values);
```

Reply:
Ack